app/presenters/publishing_api/historical_account_presenter.rb

```diff
@@ -60,10 +60,7 @@ def related_pms
       return [] unless all_appointees.include?(person_to_present)
 
       neighbouring_role_holders(all_appointees, person_to_present).map do |person|
-        {
-          "title" => person.name,
-          "base_path" => person.historical_accounts.present? ? person.historical_accounts.for_role(role.id).first.public_path : HistoricalAccountsIndexPresenter.base_path,
-        }
+        person.historical_accounts.present? ? person.historical_accounts.for_role(role.id).first.content_id : HistoricalAccountsIndexPresenter.content_id
```

Comment:
I think there's going to be an issue here - where the person has no historical account (e.g. recent PMs), link expansion isn't going to work here, since the content id will not be that of a person. Effectively, we'll not get their name. So I wonder if, for those without historical accounts, we should add them in a separate 'people without historical accounts' hash (we'd also need to make sure their roles were expanded so we could get dates in office when sorting - or just add an assumption into the frontend that anybody without a historical account is the most recent?). 

Sorry for the spanner, this bit of functionality seems ever more of a pain especially since I imagine it's barely used (we could also not fall victim to sunk cost fallacy and consider scrapping it and saying it's not worth the dev time it'd take to do this and faff around with it on the frontend too 🙈)

Maybe we can discuss as devs, there are other alternatives to solving but I increasingly wonder if it's worth it.